### PR TITLE
Add changelog for v0.2.0

### LINKED ISSUE #116: Version 0.2.0 release checklist
- [x] Update README (#118)
- [x] Add a changelog (#133)
    - [x] Suggest that any user with `local:`, `local:a:b` etc rename their keys

### PR DESCRIPTION
**Pull request by wupr**
Closes #116

### CODE REVIEW

**Comment by wupr** [comment]
@alexrutar We are missing documentation on the config file. Could you add some in README?

**Comment by alexrutar** [comment]
> @alexrutar We are missing documentation on the config file. Could you add some in README?

Will do!

File: docs/examples/default_config.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Actions to perform when adding a new item to the database from a remote provider.
+# These actions are not run for local operations, such as `autobib local`.
+[on_insert]
+
+# Whether or not to normalize whitespace, which means deleting leading and trailing
+# whitespace and converting any consecutive blocks of unescaped whitespace (such as
+# normal spaces, or newlines, or Unicode whitespace) into a single ASCII space.
+normalize_whitespace = false
+
+# A list BibTeX fields of from which to automatically set the `eprint` and
```
Comment (alexrutar) [inline docs/examples/default_config.toml:10]: I left a typo here, 'of from which'

Comment (alexrutar) [inline docs/examples/default_config.toml:10]: Spotted a typo, 'of from which'

Comment (wupr) [reply docs/examples/default_config.toml]: Thanks! Fixed.

**Comment by alexrutar** [comment]
Sounds good r.e. v0.2.0! Should we bump the version in this PR as well? inside `config.toml`

**Pull request merged**